LottieSample/src/main/kotlin/com/airbnb/lottie/samples/LottieApplication.kt

```diff
@@ -1,15 +1,25 @@
 package com.airbnb.lottie.samples
 
 import android.support.multidex.MultiDexApplication
+import android.util.Log
 import com.google.gson.FieldNamingPolicy
 import com.google.gson.GsonBuilder
-import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
 
 class LottieApplication : MultiDexApplication() {
-    val okHttpClient by lazy { OkHttpClient.Builder().build() }
+    // print HTTP protocol content
+    private val loggingInterceptor = HttpLoggingInterceptor(HttpLoggingInterceptor.Logger { Log.i("Lottie", it) }).apply { level = HttpLoggingInterceptor.Level.BODY }
+
+    val okHttpClient by lazy {
+        OkHttpClient.Builder()
+            .addInterceptor(loggingInterceptor)
+            .connectTimeout(30, TimeUnit.SECONDS)
```

Comment:
I set a connectTimeout, because some country visit [https://www.lottiefiles.com](https://www.lottiefiles.com) spent a lot of  time.